youtube2zim/playlists/scraper.py

```diff
@@ -145,9 +143,10 @@ def run_playlist_zim(self, playlist):
 
         # ensure we supplied a name
         if not has_argument("name", args):
```

Comment:
We have already checked the self.playlists_name in line 136. So, why not fail instead of checking self.playlists_name again?

Reply:
OK you're right. I think we shall remove this check completely but also fix the entrypoint to fail on missing `--playlists-name` (was changed to pass if --metadata-from was provided).
Pushing that now